src/cxx_supportlib/StrIntTools/StrIntUtils.cpp

```diff
@@ -61,6 +61,15 @@ startsWith(const StaticString &str, const StaticString &substr) {
 	}
 }
 
+bool
+endsWith(const StaticString &str, const StaticString &substr) {
+	if (str.size() >= substr.size()) {
+		return strncmp(str.c_str() + str.size() - substr.size(), substr.c_str(), substr.size()) == 0;
```

Comment:
This should use memcmp() instead of strncmp() unless you have a reason to. To be consistent with how startsWith() works.